sdks/python/apache_beam/runners/interactive/interactive_runner.py

```diff
@@ -52,7 +52,8 @@ def __init__(self,
                cache_dir=None,
                cache_format='text',
                render_option=None,
-               skip_display=False):
+               skip_display=False,
```

Comment:
This is added by an internal Googler and their code in Google3 need/set this value. So let's probably leave it be since we cannot make an atomic change that doesn't break things in Google3 at some point. I'll contact the author when we've settled down our changes.

Reply:
Ah gotcha, np then